templates/common/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "scripts": {
+      "test:watch": "{commandType} test -- --watch",
+      "test": "jest -c ./config/jest/jest.config.js",
+      "generate": "plop --plopfile generators/index.js && {commandType} pretty-quick",
```

Comment:
Lets start using our component generator here and delete the generator specific things.